install_config/configuring_aws.adoc

```diff
@@ -34,9 +32,44 @@ Zone = us-east-1c <1>
 <1> This is the Availability Zone of your AWS Instance and where your EBS Volume
 resides; this information is obtained from the AWS Managment Console.
 ====
-
+ 
 [[aws-configuring-masters]]
-== Configuring Masters
+== Configuring AWS for {product-title} Masters
+
+You can set AWS configuration on your {product-title} masters two ways:
+
+- xref:aws-configuring-masters-ansible[using Ansible and the advanced installation tool]
+- xref:aws-configuring-masters-manually[manually by modifying the *_master-config.yaml_* file]
+
+[[aws-configuring-masters-ansible]]
+=== Configuring AWS for {product-title} with Ansible
+
+During
+xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[advanced installations], 
+AWS can be configured using 
+xref:../install_config/install/advanced_install.html#advanced-install-configuring-global-proxy[the `*openshift_cloudprovider_aws_access_key*`, `*openshift_cloudprovider_aws_secret_key*`, and `*openshift_cloudprovider_kind*` parameters], which are configurable in the inventory file. 
+
+.Example AWS Configuration with Ansible
+====
+----
+# Cloud Provider Configuration
+#
+# Note: You may make use of environment variables rather than store
+# sensitive configuration within the ansible inventory.
+# For example:
+#openshift_cloudprovider_aws_access_key="{{ lookup('env','AWS_ACCESS_KEY_ID') }}"
+#openshift_cloudprovider_aws_secret_key="{{ lookup('env','AWS_SECRET_ACCESS_KEY') }}"
+#
+# AWS
+#openshift_cloudprovider_kind=aws
+# Note: IAM profiles may be used instead of storing API credentials on disk.
+#openshift_cloudprovider_aws_access_key=aws_access_key_id
+#openshift_cloudprovider_aws_secret_key=aws_secret_access_key
+----
+====
+
+[[aws-configuring-masters-manually]]
+=== Manually Configuring AWS for {product-title} Masters
```

Comment:
We may want a note that explains that the Ansible configs will create the aforementioned **/etc/aws/aws.conf** file for users and properly set the **/etc/origin/master/master-config.yaml**, **/etc/origin/node/node-config.yaml**, **/etc/sysconfig/atomic-openshift-master** and **/etc/sysconfig/atomic-openshift-node** up for users, for them.